Cork/Views/Sidebar/Components/Sidebar Package Row.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Sidebar Package Row.swift
+//  Cork
+//
+//  Created by Seb Jachec on 08/09/2023.
+//
+
+import SwiftUI
+
+struct SidebarPackageRow: View {
+
+    let package: BrewPackage
+
+    @AppStorage("allowMoreCompleteUninstallations") var allowMoreCompleteUninstallations: Bool = false
+
+    @EnvironmentObject var appState: AppState
+    @EnvironmentObject var brewData: BrewDataStorage
+    @EnvironmentObject var outdatedPackageTracker: OutdatedPackageTracker
+
+    var body: some View {
```

Comment:
The contents of this body was directly taken from the casks section (which was also identical to the row views in the formulae section)